### PR TITLE
RIA-8273 Print and send hearing bundle - WA TASK

### PR DESCRIPTION
**Pull request by javierhmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-8273

### Change description ###

 Print and send hearing bundle - WA TASK

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/wa-task-initiation-ia-asylum.dmn [hunk 1]
```diff
@@ -3807,6 +3807,65 @@
           <text>"caseProgression"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0jvkz7e">
+        <inputEntry id="UnaryTests_1xlvoo7">
+          <text>"generateHearingBundle"</text>
```
Comment (ThomasKKC) [inline src/main/resources/wa-task-initiation-ia-asylum.dmn:3812]: I think asyncStitchingComplete is the event to use for this scenario.

File: src/main/resources/wa-task-initiation-ia-asylum.dmn [hunk 2]
```diff
@@ -3807,6 +3807,65 @@
...
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1ftsn80">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0tsvny3">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0vudg83">
+          <text>true</text>
```
Comment (ThomasKKC) [inline src/main/resources/wa-task-initiation-ia-asylum.dmn:3854]: Need to publish this isAdmin too.

**Review by ThomasKKC — CHANGES_REQUESTED** [review]

File: src/main/resources/wa-task-initiation-ia-asylum.dmn [hunk 3]
```diff
@@ -3866,6 +3866,65 @@
           <text>"caseProgression"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0jvkz7e">
+        <inputEntry id="UnaryTests_1xlvoo7">
+          <text>"asyncStitchingComplete"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0zet58l">
+          <text>"finalBundling"</text>
```
Comment (ThomasKKC) [inline src/main/resources/wa-task-initiation-ia-asylum.dmn:3874]: We need to post event state, not pre event state. Please check AdvancedFinalBundlingStateHandler

**Review by ThomasKKC — CHANGES_REQUESTED** [review]

**Review by ThomasKKC — APPROVED** [review]

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Pull request merged**